tests/webview/integration/test_webview_integration.py

```diff
@@ -0,0 +1,22 @@
+import requests
+
+from tests import markers
+
+
+@markers.webview
+@markers.nondestructive
+@markers.parametrize("uuid, version", [("fea3130c-6e57-41b2-a00f-0267ffae273c", 5)])
+def test_archive_redirect_for_google_results(archive_base_url, uuid, version):
+    # end-to-end integration test for redirecting google away from archive URLs
+    # see: https://github.com/openstax/cnx/issues/209
+
+    # GIVEN an "archive" URL
+    url = f"https://archive-qa.cnx.org/contents/{uuid}@{version}"
```

Comment:
Since this is now in staging can we go ahead and use the `archive_base_url` fixture?

Reply:
`https://{archive_base_url}/contents/{uuid}@{version}`

Reply:
It is in staging?? It's _just_ been merged: https://github.com/openstax/cnx-deploy/pull/1084

Reply:
plz update the f-string to use the `archive_base_url`.

Reply:
Done.